node/modules/lp2p/rcmgr.go

```diff
@@ -19,15 +19,11 @@ import (
 
 var log = logging.Logger("rcmgr")
 
-func NewDefaultLimiter() *rcmgr.BasicLimiter {
-	return rcmgr.NewDefaultLimiter()
-}
-
-func NewResourceManager(limiter *rcmgr.BasicLimiter) (network.ResourceManager, error) {
+func NewResourceManager(limiter *rcmgr.Limiter) (network.ResourceManager, error) {
 	var opts []rcmgr.Option
-	libp2p.SetDefaultServiceLimits(limiter)
+	libp2p.SetDefaultServiceLimits(&rcmgr.DefaultLimits)
```

Comment:
still need to figure out what's going on here with the limit configs. not sure if we're trying to pull libp2p's defaults or use our own

Reply:
using our own